lib/class-wp-rest-plugins-controller.php

```diff
@@ -0,0 +1,829 @@
+<?php
```

Comment:
What about mu-plugins?

Reply:
I don't think we should support mu-plugins or drop-ins at the moment. They can be added at a later point.